### ISSUE TITLE
Connection to Atlas

### ISSUE THREAD

**Issue by stefanwalther** [jeromelebel/MongoHub-Mac#264]
Did anybody succeed to connect to a MongoDB instance on Atlas?
I have tried several variants, but just do not succeed (but e.g. connecting with MongoDB Compass Community edition works from my local Mac).

**Comment by kayschmitt** [comment]
![Screen Shot 2020-01-30 at 2 57 35 PM](https://user-images.githubusercontent.com/12105027/73490001-3bb05a00-4371-11ea-9be4-0387a3514520.png)


@stefanwalther I successfully connected to an atlas instance. 

The main problem I was having when trying to connect was that I was using the name of the database I wanted to connect to as the "Database" when you need to use "admin" regardless 
You'll get the list of servers and the set name from the "Command Line Tools" in atlas under the "Binary Import and Export Tools" section. 
Here's what my servers look like since it got cut off in the screenshot:
nameofcluster-shard-00-00-vm0ag.mongodb.net:27017,nameofcluster-shard-00-01-vm0ag.mongodb.net:27017,nameofcluster-shard-00-02-vm0ag.mongodb.net:27017

**Comment by Anaphase** [comment]
Yeah I can't seem to connect to Atlas either, even if I specify "admin" as the database as @kayschmitt suggested. All I get is a "no acceptable peer could be found." error when I set it up just like the screenshot above (of course replacing my actual cluster name and credentials.)
